src/guides/v2.3/extension-dev-guide/admin-grid.md

```diff
@@ -377,6 +377,15 @@ class Collection extends SearchResult
 
 It uses a custom collection file to add custom filters to map, and makes the grid filters work with the ID and name fields. Without `addFilterToMap`, you will not be able to search within the `name` column.
 
+The resource model class translates into `app/code/Dev/Grid/Model/ResourceModel/Category.php`:
```

Comment:
```suggestion
The resource model class translates to `app/code/Dev/Grid/Model/ResourceModel/Category.php`:
```

Reply:
This line is copied, not mine :D Right above is the original sentence, which is `The collection class translates into app/code/Dev/Grid/Ui/DataProvider/Category/Listing/Collection.php`

Do you want me to change it as well?

Reply:
yes please - "translates to" is better English than "translates into"